### DISCUSSION TITLE
How to output dxf data format instead of data stream

### DISCUSSION THREAD

**Discussion by zhongqingbin** [mozman/ezdxf#616] [Q&A]
How to output dxf data format instead of data stream
The current format is
![image](https://user-images.githubusercontent.com/7828474/149749185-060d9211-769e-4597-9301-bb30e3488c2c.png)
The data format I want is
![image](https://user-images.githubusercontent.com/7828474/149749286-d7ea779d-edee-4121-a64d-cd4a9d57d99e.png)

**Comment by chibai** [comment]
This is more like a  discussion rather than issue.
If you want to analyze the file yourself,  You can learn tricks to do this in https://github.com/mozman/ezdxf/blob/master/src/ezdxf/recover.py
or you can read the file by ezdxf and output it in specific format.
https://ezdxf.readthedocs.io/en/stable/sections/index.html
